Neos.Flow/Documentation/conf.py

```diff
@@ -59,9 +59,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = 'dev-master'
+version = '7.0'
 # The full version, including alpha/beta/rc tags.
-release = 'dev-master'
+release = 'dev-7.0.x'
```

Comment:
```suggestion
release = '7.0.0'
```

Yes, that never get's updated during the lifecycle (yet), but still…

Reply:
Hm… actually that's what I did in https://github.com/neos/flow-development-collection/commit/37396baf4725379f7d24146738fe17f1b8a30d4f. 🙈

Probably _because_ it is not updated automagically for releases and rendered from the branch tip anyway. All is well!